spring-boot-dependencies/pom.xml

```diff
@@ -2472,6 +2473,11 @@
 				<artifactId>xml-apis</artifactId>
 				<version>${xml-apis.version}</version>
 			</dependency>
+			<dependency>
```

Comment:
Could you move this up a bit please? We try to keep the entries in alphabetical order using `groupId` first and then `artifactId` if required